test/test-eth-trezor-keyring.js

```diff
@@ -128,16 +132,22 @@ describe('TrezorKeyring', function () {
     })
 
     it('should call TrezorConnect.getPublicKey if we dont have a public key', async function () {
+      const getPublicKeyStub = sinon.stub(TrezorConnect, 'getPublicKey').callsFake(
+        () => Promise.resolve({}),
+      )
       chai.spy.on(TrezorConnect, 'getPublicKey')
```

Comment:
Nit: Is this spy redundant now? :thinking: It's already a stub, which implements the sinon spy API as well. This is double-spying.

Reply:
The other tests have redundant spies now as well I noticed.